src/pkg/caendr/caendr/services/cloud/lifesciences.py

```diff
@@ -47,36 +46,6 @@ def start_pipeline(SERVICE, task_id, pipeline_request):
     raise PipelineRunError(err)
 
 
-def create_pipeline_operation_record(task, response):
```

Comment:
Moved to `GCPCloudRunRunner.run` method. The `PipelineOperation` execution record is now handled implicitly by that class.